### DISCUSSION TITLE
<create a new repository>

### DISCUSSION THREAD

**Discussion by ahmadali-jamali** [community/community#46150] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hey, guides. I am new to Git Hub. I have made 3 repositories before, but today when I want to create a new one, the green button of "create new repository" does not let me create a new one(without explaining anything) the green button does not light. Are any restrictions? what should I do?
 thanks.

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @ahmadali-jamali 

Try directly via URL for create new repo:

https://github.com/new

**Reply by ahmadali-jamali** [reply]
Thanks for your reply. I have opened it with Firefox Browser and it is fixed. I do not know why it happened in chorome

**Reply by GeorgeKontsevik** [reply]
Restart browser (close and open again) worked for me

**Comment by ghostinhershell** [comment]
Hey there! 👋

Thanks for posting in the GitHub Community, @ahmadali-jamali! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, it sounds like your issue may be solved by clearing your cache and cookies. Here's an article from Google on how to do that in the Chrome browser: https://support.google.com/accounts/answer/32050?hl=en&co=GENIE.Platform%3DDesktop.

I've gone ahead and moved this to the correct category for you. Good luck!
